README.md

```diff
@@ -85,3 +85,8 @@ Repository exists already under https://github.com/modularity-modules/:
 | `redis` | | | | | | |
 |  `ruby`  |   |   |   |   |   | [**2017-10-03**](https://mbs.fedoraproject.org/module-build-service/1/module-builds/1036) |
 |  `varnish`  |   |   |   |   |   | [**2017-07-10**](https://mbs.fedoraproject.org/module-build-service/1/module-builds/703) |
+| `golang` | lsm5 | | | | | |
+| `container-shared` | lsm5 | | | | | |
```

Comment:
not sure if it is good name...

Reply:
suggestions?

Reply:
i'm open to anything that doesn't have the word `docker` in it :) , since `cri-o` and `buildah` will need it too

Reply:
`base-container`, `container-platform`, `platform-container` ????

Reply:
@ignatenkobrain what's wrong w/ the name? Please let us know if you have another idea.

Thanks,

Reply:
it's hard to say what name should be.... because I don't know what content should be there =)

I guess those are some golang packages (i.e. sources) in which case you probably don't even need to create module, you can just include packages directly in golang/docker/cri-o/buildah since golang links statically.

Reply:
@ignatenkobrain so the packages in container-shared are basically runtime/install time deps for cri-o and docker. That's why those were in a separate module. Those aren't build deps for docker or cri-o.

Reply:
@lsm5 can you tell which packages are there? =) I don't see them in this PR.

Reply:
@ignatenkobrain see: https://github.com/lsm5/container-shared-module/blob/master/container-shared.yaml

Reply:
in that case probably `container-runtime`... although I'm not sure why you want to have atomic in there... Or something related to OCI. probably @contyk has better ideas

Reply:
maybe `container-utils`?

Reply:
Yay, bikeshedding about the name ;) Well, do you expect the module to be usable on its own or only together with one of the other container modules? If it needs something else, I think `shared` is a reasonable name and doesn't evoke the feeling that installing this module is enough to get everything you need, like the other suggested names do.

Reply:
"runtime" has become a loaded term in the container space. I'd like to avoid it altogether with this module's name. I think I'm leaning towards container-tools or container-utils.

@lsm5 should buildah be included in this module as well?

Reply:
@mrguitar @rhatdan mentioned this as a separate module in prior discussion, though I'm ok either way. buildah doesn't depend on anything in docker or cri-o so including it in container-{shared|utils|tools] should be fine.

Reply:
@TomasTomecek apache-commons is named like that because it is how it is named in upstream ;)

container-tools would be better possibly.

Reply:
seems like everyone agrees with container-tools, so I'll send a new commit to reflect that, also gonna get rid of buildah module